src/gadgets.rs

```diff
@@ -1,5 +1,6 @@
-pub mod bloom_filter;
-pub mod hash;
-pub mod response_accumulator;
+mod bits2num;
+mod bloom_filter;
+mod hash;
```

Comment:
Cool! The reason I made it `pub` was that it's included in the documentation. But we can use `cargo doc --document-private-items`, so that's a better solution! 🎉